src/test/common/steps/navigation.js

```diff
@@ -21,7 +21,7 @@ const ENTER_NAME_PAGE = 'enter name'
 const DO_YOU_LIVE_IN_SCOTLAND_PAGE = 'do you live in Scotland'
 const ENTER_NINO_PAGE = 'enter national insurance'
 const ENTER_DOB_PAGE = 'enter date of birth'
-const DO_YOU_HAVE_CHILDREN_THREE_OR_YOUNGER_PAGE = 'do you have children three or younger'
+const DO_YOU_HAVE_CHILDREN_THREE_OR_YOUNGER_PAGE = 'do you have children under four years old'
```

Comment:
Should this constant be renamed too, or is that the next PR?

Reply:
Its in the next PR